tinygrad/renderer/assembly.py

```diff
@@ -82,15 +83,14 @@ def render_val(x, dtype):
 ])
 
 class PTXRenderer(Renderer):
-  device = "CUDA"
   suffix = "PTX"
-  global_max = (2147483647, 65535, 65535)
-  local_max = (1024, 1024, 64)
-  shared_max = 49152
   tensor_cores = [tc for tc in CUDARenderer.tensor_cores if tc.dtype_in == dtypes.half]
   code_for_op = asm_for_op
   extra_matcher = ptx_matcher
-  def __init__(self, arch:str, device="CUDA"): self.device, self.tensor_cores = device, PTXRenderer.tensor_cores if int(arch[3:]) >= 80 else []
+  def __init__(self, arch:str, device="CUDA"):
+    self.device = device
+    self.global_max, self.local_max, self.shared_max = CUDARenderer.global_max, CUDARenderer.local_max, CUDARenderer.shared_max
```

Comment:
Err, do these change? Why aren't they global scope?

Reply:
They don't, you are right, they should be global scope